reconcile/resource_template_tester.py

```diff
@@ -11,13 +11,13 @@
 from reconcile.utils import gql
 from reconcile.utils.semver_helper import make_semver
 
-QONTRACT_INTEGRATION = "template-tester"
+QONTRACT_INTEGRATION = "resource-template-tester"
 QONTRACT_INTEGRATION_VERSION = make_semver(0, 1, 0)
 
 
 TEMPLATE_TESTS_QUERY = """
 {
-  tests: template_tests_v1 {
+  tests: resource_template_tests_v1 {
```

Comment:
missing gql schema sync?

Reply:
Yes, it's an old-school integration.

Reply:
I mean there is no `make gql-introspection` change for new schema?

Reply:
Oh, there is. it's not required since that validation is not used here. I can add it.